qiskit_aer/backends/aerbackend.py

```diff
@@ -209,7 +256,7 @@ def run(self,
         if isinstance(experiments, list):
             aer_job = AerJobSet(self, job_id, self._run, experiments, executor)
         else:
-            aer_job = AerJob(self, job_id, self._run, experiments, executor)
+            aer_job = AerJob(self, job_id, self._run, qobj=experiments, executor=executor)
```

Comment:
This is a pre-existing bug so we don't need to fix it here, but for the case where circuits or schedules are passed to `run()` we probably want to be storing the input circuit objects on the job and not the qobj. Since for most people the qobj isn't a meaningful interface.